.github/workflows/continuous-integration.yml

```diff
@@ -118,3 +118,31 @@ jobs:
         run: yarn --immutable
       - name: Validate dynamic side effects
         run: yarn build:dynamic-side-effects -- --check
+
+  Check_PackageIndexFiles:
+    name: Check Package Index Files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+      - name: Use Node.js v20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: yarn
+          registry-url: https://registry.yarnpkg.org/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Generate index files
+        run: yarn gen-index --parallel --concurrency $(ls packages | wc -l) --output-logs new-only
+      - name: Test for differences
+        run: |
+          git add .;
+          if git diff-index --quiet HEAD --; then
+            echo "No index discrepancies found, exiting with code 0";
+            exit 0;
+          else
+            echo "Index discrepancies found, displaying and exiting with code 1";
+            git diff;
```

Comment:
gotta fix this one since it doesn't display

Reply:
Try writing it to a file then setting error similar as its done here: https://github.com/skyra-project/discord-components/blob/70263974e2950a9c0c67ca803546bd2fc880bf88/.github/workflows/generated-file-validations.yml#L31

Failed run example: https://github.com/skyra-project/discord-components/actions/runs/7861891265